rasa/core/policies/memoization.py

```diff
@@ -223,18 +230,14 @@ def _prediction_result(
         return result
 
     def predict_action_probabilities(
-        self,
-        tracker: DialogueStateTracker,
-        domain: Domain,
-        precomputations: Optional[MessageContainerForCoreFeaturization] = None,
```

Comment:
`tests/core/test_policies.py` will fail without this - it assumes all `predict_action_probabilities` look alike

Reply:
ah, I see, you just removed that parameter everywhere in the tests :)